hieradata/org/lsst.yaml

```diff
@@ -193,3 +196,7 @@ yum::plugin::versionlock:
   clean: true
 
 letsencrypt::email: "rubinobs-it-las@lsst.org"
+profile::core::icinga_agent::install: true
+profile::core::icinga_agent::host_template: "GeneralHostTemplate"
+profile::core::icinga_agent::icinga_master_fqdn: "icinga-master.ls.lsst.org"
```

Comment:
Site specific configuration, such as the name of the icinga master, needs to go into the site specific layer. Presumably `hieradata/site/ls.yaml`.

Reply:
moved them here based on your previous suggestion of handle icinga at a org level. Moving them back.